docs/kb/semgrep-cloud-platform/missing-pr-commments.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Use this reference to check why you may not be receiving Semgrep comments on PRs/MRs.
+tags:
+  - Semgrep CI
+  - PR Comments
+  - Rule Board
+---
+
+# Why am I not receiving PR/MR comments?
+
+If you have configured Semgrep in CI and Semgrep Cloud Platform to create comments when a rule generates a finding in a PR or MR, but you are not seeing those comments, check the following possibilities.
+
+## Are comments supported on your SCM?
+
+PR/MR Comments are currently supported for GitHub, GitLab, and Bitbucket. If you are using another SCM, comments are not yet supported. Bitbucket Data Center is also not supported yet.
+
+If you are using a self-hosted version of GitHub (GitHub Enterprise) or GitLab (GitLab Self-Managed), see [Integrating Semgrep Cloud Platform with self-hosted enterprise repositories](/docs/semgrep-cloud-platform/scm/) for more details on configuration.
+
+## Have you configured permissions and tokens correctly?
+
+### GitHub
+
+GitHub relies on the Semgrep GitHub app to make comments on code. To receive comments on a project, you must [onboard](/docs/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) the project to Semgrep Cloud Platform, or ensure that your GitHub app is configured with permissions for all repositories. See [Enabling GitHub pull request comments](/docs/semgrep-cloud-platform/github-pr-comments/) for details.
```

Comment:
```suggestion
GitHub relies on the Semgrep GitHub app to make comments on code. To receive comments on a project, ensure that you have performed the following steps:

* You have [onboarded](/docs/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) the project to Semgrep Cloud Platform.
* You have configured your GitHub app with permissions for all repositories that are scanned by Semgrep Cloud Platform. See [Enabling GitHub pull request comments](/docs/semgrep-cloud-platform/github-pr-comments/) for details.
```

Just making it consistent with the list below.
Also (I am so sorry) GSG prefers a more explicit style of [writing introductory paragraphs to procedures](https://developers.google.com/style/procedures), so I suggested a complete sentence.